src/ol_infrastructure/applications/mitopen/__main__.py

```diff
@@ -794,6 +794,20 @@
     ),
     opts=ResourceOptions(provider=github_provider),
 )
+gh_workflow_fastly_api_key_env_secret = github.ActionsSecret(
+    f"ol_mitopen_gh_workflow_fastly_api_key_env_secret-{stack_info.env_suffix}",
+    repository=gh_repo.name,
+    secret_name="FASTLY_API_KEY",  # noqa: S106 # pragma: allowlist secret
```

Comment:
These need to have `env_var_suffix` appended to the end like the secrets above otherwise both stacks are going to smash on the same secret name.

Reply:
I generated the key to work for both RC and production, but I updated the stack id to include the suffix.